### PR TITLE
wip: Create WELCOME_TO_IPFS.md

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

**Comment by diasdavid** [comment]
Somethings to be included (these are coming from other onboarding lists I've written, they need to be reworded to the audience of this repo):

- The js-ipfs Core Dev Team tracks their work using a [Waffle board](http://waffle.io/ipfs/js-ipfs). Learn how work gets tracked at [MGMT.md](https://github.com/ipfs/js-ipfs/blob/master/MGMT.md) and [Weekly Sync issue](https://github.com/ipfs/js-ipfs/issues/1179).
- The IPFS development team hangs out on IRC on the #ipfs-dev channel @freenode. I strongly recommend you to join us there :)
- Make sure to read the IPFS community [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
- To get yourself more familiar, make sure to:
  - Run through all the Tutorials and Examples to get yourself more familiar https://github.com/ipfs/js-ipfs/tree/master/examples
  - Read the CRDT Tutorial - https://blog.ipfs.io/30-js-ipfs-crdts.md
  - Read the PubSub Tutorial - https://blog.ipfs.io/29-js-ipfs-pubsub/
  - Watch Juan’s IPFS Stanford Talk for a comprensive overview of the Protocol https://www.youtube.com/watch?v=HUVmypx9HGI

File: README.md [hunk 1]
```diff
@@ -34,9 +34,11 @@ IPFS is a distributed file system that seeks to connect all computing devices wi
 
 IPFS is becoming a new major subsystem of the internet. If built right, it could complement or replace HTTP. It could complement or replace even more. It sounds crazy. It _is_ crazy.
 
-[![Alpha Demo](img/ipfs-alpha-video.png)](https://www.youtube.com/watch?v=8CMxDNuuAiQ)
+You can watch the following talks to get up to speed on what IPFS is and what it does:
 
-Want to see more? Check out Juan Benet's talk at Sourcegraph - [IPFS: The Permanent Web](https://www.youtube.com/watch?v=Fa4pckodM9g).
+| Alpha release demo | IPFS talk at Sourcegraph | Stanford Seminar |
+|:---:|:---:|:---:|
+| [![](img/alpha-demo.jpg)](https://youtu.be/8CMxDNuuAiQ) | [![](img/talk-sourcegraph.jpg)](https://www.youtube.com/watch?v=Fa4pckodM9g) | [![](img/stanford-seminar.jpg)](https://youtu.be/HUVmypx9HGI)
```
Comment (diasdavid) [inline README.md]: Sizes need to be adjusted
image

**Comment by diasdavid** [comment]
- Jeromy Coffee Talks - Bitswap - https://www.youtube.com/watch?v=9UjqJTCg_h4
- Jeromy Coffee Talks - Files API - https://www.youtube.com/watch?v=FX_AXNDsZ9k
- Jeromy Coffee Talks - gx - https://youtu.be/dL9siwm03H4

**Comment by diasdavid** [comment]
Hi friends! I started this PR after receiving countless requests for better documentation. My goal is to create the discussion and the gathering of good sources (i.e talks, tutorials, blog posts) to learn about IPFS so that it gets simpler to ramp up new contributors.

I do agree that there is a lot of room to improve and that a lot of knowledge is trapped in some people heads. However, given the lack of availability and the uncertainty of which parts are poorly documented, I come here to ask everyone to help us understand what is missing by giving __thorough reviews of the current IPFS specs and docs__ and __open issues with your questions__. Then once you have your questions answered, those answers should be converted back to docs+specs.

@mr0grog is this something you would like to lead?

**Comment by diasdavid** [comment]
Listing here the issues containing feedback from new users that climbed the whole IPFS mountain:

- https://github.com/ipfs/specs/issues/145
- https://github.com/ipfs/js-ipfs/issues/1416
- https://github.com/ipfs/notes/issues/289

**Comment by jessicaschilling** [comment]
Closing (stale).

**Pull request closed without merging**